.make/Makefile.mk

```diff
@@ -49,6 +50,7 @@ pre-build:
 post-build:
 
 pre-push:
+	docker login -u ci-cd -p ci-cd-pipeline-8237 nexus.engageska-portugal.pt
```

Comment:
Shall use Environment variables instead:

- **CI_REGISTRY_PASS_LOGIN** instead of `ci-cd-pipeline-8237`
- **CI_NX_REGISTRY** instead of `nexus.engageska-portugal.pt`

Reply:
Since this change was giving an error, I have kept it unchanged.